CRM/Price/BAO/PriceField.php

```diff
@@ -351,9 +351,9 @@ public static function addQuickFormElement(
         }
         else {
           $message = ts('%1 must be a number (with or without decimals).', [1 => $label]);
-          $type = 'numeric';
+          $type = 'money';
```

Comment:
This now says `if ($isQuickConfig) $type = 'money'; else $type = 'money';`. Would it be useful to declare `$type` outside the if-statement, or just hardcode it in the `addRule` call?